extensions/sql-database-projects/src/dialogs/addDatabaseReferenceDialog.ts

```diff
@@ -48,7 +48,10 @@ export class AddDatabaseReferenceDialog {
 	public exampleUsage: azdataType.TextComponent | undefined;
 	private projectRadioButton: azdataType.RadioButtonComponent | undefined;
 	private systemDatabaseRadioButton: azdataType.RadioButtonComponent | undefined;
-
+	private systemDatabaseArtifactRefRadioButton: azdataType.RadioButtonComponent | undefined;
+	private systemDatabasePackageRefRadioButton: azdataType.RadioButtonComponent | undefined;
+	private systemDbRefRadioButtonsComponent: azdataType.FormComponent | undefined;
+	private systemDbRefStyle: SystemDbReferenceStyle = SystemDbReferenceStyle.ArtifactReference;
```

Comment:
I decided to use "Style" here in the variable name and type name instead of "Type" because there was already a ReferenceType being used in this file, but happy to change it if there are other ideas :)

Reply:
I'm good with "style"

Reply:
Good idea! Sent a separate PR https://github.com/microsoft/azuredatastudio/pull/23347 to update "referenceType" in ADS/VSCode to "referencedDatabaseType" so this PR can get that update after that's merged to main.

Reply:
Sounds good!  I signed off on that PR, and will pop back to this one once it's merged in.

Reply:
Updated!